xilem_web/src/concurrent/memoized_await.rs

```diff
@@ -19,10 +33,27 @@ pub struct MemoizedAwait<State, Action, OA, InitFuture, Data, Callback, F, FOut>
     phantom: PhantomData<fn() -> (State, Action, OA, F, FOut)>,
 }
 
+impl<State, Action, OA, InitFuture, Data, Callback, F, FOut>
+    MemoizedInner<State, Action, OA, InitFuture, Data, Callback, F, FOut>
+where
+    FOut: fmt::Debug + 'static,
+    InitFuture: Fn(&Data) -> F,
+{
+    fn debounce_ms(mut self, milliseconds: usize) -> Self {
+        self.debounce_ms = milliseconds;
+        self
+    }
+
+    fn reset_debounce_on_update(mut self, reset: bool) -> Self {
+        self.reset_debounce_on_update = reset;
+        self
+    }
+}
+
```

Comment:
I think we can omit this, I think it's just boilerplate without bringing value, but no strong feelings here either.